packages/orbit-components/src/Itinerary/ItinerarySegment/ItinerarySegmentStop/index.d.ts

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+
+/** DOCS:
+  ItineraryPartPlace is an atomic unit of the Itinerary component, shows two locations, date and time,
+  has warning property which changes the icon to `<AlertCircle color="warning" />`
+  to attract user attention about some important information about journey.
+*/
+export interface Props {
+  /** The date of ItineraryPartPlace */
+  readonly date: string;
+  /** The time of ItineraryPartPlace */
+  readonly time: string;
+  /** The city of ItineraryPartPlace */
+  readonly city: string;
+  /** The place of ItineraryPartPlace */
+  readonly station: string;
+  /** ItineraryPartPlace which status is HiddenCity */
+  readonly hidden?: boolean;
+  /** Changes the icon of current ItineraryPartPlace, if there is some important information for user */
+  readonly warning?: boolean;
```

Comment:
Orange doesn't seem to be the only possibility, I see the blue one as well:

<img width="413" alt="Screen Shot 2021-11-11 at 22 10 13" src="https://user-images.githubusercontent.com/471278/141369415-6839d546-2d26-43aa-884a-4bb602d3b2c1.png">

Instead of adding a flag for whether it's warning or not, I recommend adding a `type` prop instead to allow for multiple types.